plugins/modules/rds_cluster.py

```diff
@@ -907,18 +938,22 @@ def get_restore_s3_options(params_dict):
         "BacktrackWindow",
         "BackupRetentionPeriod",
         "CharacterSetName",
+        "DatabaseInsightsMode",
```

Comment:
None of these options are supported by https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds/client/restore_db_cluster_from_s3.html#RDS.Client.restore_db_cluster_from_s3. These attributes could be modified after restoring the cluster.